src/Argu/project.json

```diff
@@ -0,0 +1,65 @@
+{
+    "version": "3.0.0-beta02",
+    "buildOptions": {
+        "debugType": "portable",
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "AssemblyInfo.fs",
+                "Types.fs",
+                "Attributes.fs",
+                "Utils.fs",
+                "ConfigReaders.fs",
+                "UnionArgInfo.fs",
+                "PreCompute.fs",
+                "UnParsers.fs",
+                "ParseResults.fs",
+                "Parsers/Common.fs",
+                "Parsers/Cli.fs",
+                "Parsers/KeyValue.fs",
+                "ArgumentParser.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+        "version": "1.0.0-preview2-*",
+        "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                    "NETSTANDARD1_5",
+                    "NETSTANDARD1_6"
+                ]
+            },
+            "dependencies": {
+                "System.Xml.XDocument": "4.0.11",
+                "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
```

Comment:
the `FSharp.Core` is really new, i'd like to wait one or two days until is tested.
And it's the same as `Microsoft.FSharp.Core.netcore` for netstandard (is merged inside).
I think when ready the `Microsoft.FSharp.Core.netcore` will become an alias (as metapackage) for `FSharp.Core`.

@ctaggart my pr i remove the .net 40 version from project.json, so no problem about that.